src/scss/components/fat-footer/_footer.scss

```diff
@@ -6,20 +6,19 @@
     @include modular-spacing('padding-top', 4);
     grid-template-areas: "A B";
 
-    // need to adjust for other sizes and adjust grid template to work with the content columns
-    @include grid-media-min(md) { // 1200px
+    @include grid-media-min('md') { // 1200px
       -ms-grid-columns: 3fr 5fr;
```

Comment:
When I use `grid-template-areas`, autoprefixer automatically add the -ms properties for me (and it would add the grid column gap as an extra column for IE). Does it not work here?

Reply:
I believe we aren't using that because it conflicted with tailwind in this theme. If you look at postcss.config.js file you'll see we don't include the grid autoprefixer.